Example/CollectionViewToolsExample/Diff/DiffViewController.swift

```diff
@@ -305,20 +319,24 @@ class DiffViewController: UIViewController {
         return sectionItem
     }
 
-    private func makeGroupTitleCellItem(group: Group) -> TextCellItem {
-        let cellItem = TextCellItem(text: group.title,
-                                    backgroundColor: group.color.uiColor,
-                                    font: .systemFont(ofSize: 16),
-                                    roundCorners: true,
-                                    contentRelatedWidth: false)
-        cellItem.diffIdentifier = "group_title_\(group.id)"
-        cellItem.itemDidSelectHandler = { [weak self] _ in
+    private func makeGroupHeaderItem(group: Group) -> HeaderViewItem {
+        let headerItem = HeaderViewItem(title: group.title,
+                                        backgroundColor: group.color.uiColor,
+                                        isFolded: group.isFolded)
+//        headerItem.diffIdentifier = "group_header_\(group.id)"
```

Comment:
Why did you leave this commented code here?

Reply:
🤷‍♂️